api/_utils/db/db-manager.ts

```diff
@@ -15,6 +14,7 @@ const {
   PG_PASSWORD: password,
   PG_HOST: host,
   PG_DATABASE: database,
+  MAX_LITER_AMOUNTS,
```

Comment:
Should the API really check for the max values? Shouldn't the API not be agnostic of that? Also if we check this we also need to check if the value is below 0?

Reply:
So you would do this only on the Frontend? I'm fine with that.